SKILL.tsv

```diff
@@ -3821,22 +3821,22 @@ SKILL_20150414_003820	Hovering
 SKILL_20150414_003821	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Float on target area and attack enemy passing below.
 SKILL_20150414_003822	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Throw practice hawk doll and attack that area.
 SKILL_20150414_003823	xxHand and Shoot
-SKILL_20150414_003824	Hang on the hawk. You do not receive damage while hanging and can make normal attacks on enemies.
-SKILL_20150414_003825	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Create magic circle that recovers HP or attacks enemy. Does not apply to enemies in air.
-SKILL_20150414_003826	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Create magic circle that heals level 1 abnormal state of allies or inflict damage on enemy. Not applied to enemies in air.
-SKILL_20150414_003827	Create magic circle that continuously decerase defense of enemy.
-SKILL_20150414_003828	Create magic circle that increases skill level once.
-SKILL_20150414_003829	Summon holy torch to increase HP recovery of nearby allies.
-SKILL_20150414_003830	Create magic circle that increase ally's critical or decrease enemy's critical.
-SKILL_20150414_003831	Increase number of beneficial buff that the party members can receive.
-SKILL_20150414_003832	{#993399}{ol}[Magic] - [Lightning]{/}{/}{nl}Use fhe powerer Thurderer for magic circle that drop lightning on the enemy.
-SKILL_20150414_003833	Make a mark on the enemy. Strength and Intelligence increase when you or your party members kill the marked enemy.
-SKILL_20150414_003834	Create magic circle that help keep beneficial buff of you and your party.
-SKILL_20150414_003835	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Spray holy water to increase defense of allies and inflict damage to enemies.
+SKILL_20150414_003824	Hang on the hawk. You do not receive damage while hanging and can make normal attacks to enemies.
+SKILL_20150414_003825	${#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Creates a magic circle that recovers HP or inflicts damage to enemies. This does not apply to airborne enemies.
+SKILL_20150414_003826	${#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Creates a magic circle that removes a Lv1 abnormal status from allies or inflicts damage to enemies. This does not apply to airborne enemies.
```

Comment:
Same "inflict" issue. Also, you use different forms of verbs between the first and the second sentence. Should be rewritten to "Creates a magic square that removes a Lv1 abnormal status from allies or inflicts damage on enemies. Does not apply to airborne enemies."